app/models/host_report.rb

```diff
@@ -56,6 +57,19 @@ def self.search_by_keyword(_key, operator, value)
     }
   end
 
+  def self.summarise(_time = 1.day.ago, *hosts, format)
```

Comment:
@lzap Not sure about time, in ConfigReport we showed reports from X time ago. 
Do we want to do this here as well or are we showing the latest host reports regardless of their time?

Reply:
This method should accept `hosts_query` which should already contain a where clause. No need to pass the time into this method. Also why do you have the splat (`*hosts`)? This should have been just an ActiveRecord relation.